docs/csharp/getting-started/with-visual-studio-code.md

```diff
@@ -56,3 +56,23 @@ If you'd like to get started with a simple "Hello World" program on .NET Core, f
 
 You can also watch a short video tutorial for further setup help on [Windows](https://channel9.msdn.com/Blogs/dotnet/Get-started-with-VS-Code-using-CSharp-and-NET-Core), [macOS](https://channel9.msdn.com/Blogs/dotnet/Get-started-with-VS-Code-using-CSharp-and-NET-Core-on-MacOS), or [Linux](https://channel9.msdn.com/Blogs/dotnet/Get-started-with-VS-Code-Csharp-dotnet-Core-Ubuntu).
 
+## Debug
+1. Open Program.cs by clicking on it. The first time you open a C# file in VS Code omnisharp will load in the editor.
+
+  ![OpenCS](media/with-visual-studio-code/OpenCS.PNG)
+2. Open the Debug tab on the left side. The icon is a bug with a cancel line through it.
+    * You can also open the Debug tab by selecting View > Debug
+
+  ![OpenDebug](media/with-visual-studio-code/OpenDebug.PNG)
+2. Locate the green arrow at the top of the pane. Make sure the drop down next to it has `.NET Core Launch (console)` selected.
+
+  ![SelectCore](media/with-visual-studio-code/SelectCore.PNG)
+3. Add a breakpoint to your project by clicking in the space left of the line numbers in the editor.
+
+  ![SetBreakpoint](media/with-visual-studio-code/SetBreakpoint.PNG)
+4. Hit F5 or the green arrow to start debugging.
```

Comment:
I'd add one sentence here to say that the debugger stops execution of your program when it reaches the breakpoint you set in (3) above.